tools/pythonpkg/src/pyexpression/initialize.cpp

```diff
@@ -0,0 +1,246 @@
+#include "duckdb_python/pybind11/pybind_wrapper.hpp"
+#include "duckdb_python/expression/pyexpression.hpp"
+#include "duckdb/common/helper.hpp"
+#include "duckdb/common/vector.hpp"
+#include "duckdb_python/python_conversion.hpp"
+
+namespace duckdb {
+
+void InitializeStaticMethods(py::module_ &m) {
+	const char *docs;
+
+	// Constant Expression
+	docs = "Create a constant expression from the provided value";
+	m.def("ConstantExpression", &DuckDBPyExpression::ConstantExpression, py::arg("value"), docs);
+
+	// ColumnRef Expression
+	docs = "Create a column reference from the provided column name";
+	m.def("ColumnExpression", &DuckDBPyExpression::ColumnExpression, py::arg("name"), docs);
+
+	// Case Expression
+	docs = "";
+	m.def("CaseExpression", &DuckDBPyExpression::CaseExpression, py::arg("condition"), py::arg("value"), docs);
+
+	// Star Expression
+	docs = "";
+	m.def("StarExpression", &DuckDBPyExpression::StarExpression, py::kw_only(), py::arg("exclude") = py::list(), docs);
+	m.def(
+	    "StarExpression", []() { return DuckDBPyExpression::StarExpression(); }, docs);
+
+	// Function Expression
+	docs = "";
+	m.def("FunctionExpression", &DuckDBPyExpression::FunctionExpression, py::arg("function_name"), docs);
+}
+
+static void InitializeDunderMethods(py::class_<DuckDBPyExpression, shared_ptr<DuckDBPyExpression>> &m) {
```

Comment:
DunderMethods?

Reply:
dunder = "double underscore", python is full with `__name__` methods and they're commonly referred to as dunder methods